test/test_core.py

```diff
@@ -437,6 +437,9 @@ def setup_esm_integration(self):
   def maybe_closure(self):
     if '--closure=1' not in self.cflags and self.should_use_closure():
       self.cflags += ['--closure=1']
+      if self.is_wasm2js():
+        # wasm2js output currently contains closure warnings
+        self.cflags += ['-Wno-closure']
```

Comment:
doesn't this mean we'd lose coverage for wasm2js+closure?

Reply:
Basically closure + wasm2js genrates warnings, and by default we run with `-Werror` in our test code.

This means that any tests that do wasm2js + closure also need to disabled closure warnings using `-Wno-closure`.  This is already done elsewhere in the tests (search for `-Wno-closure`) in test_other.py, for example.

Did you perhaps think that this line was disabling closure itself (and not just closure warnings)?